### DISCUSSION TITLE
Topic are being recreated after deletion

### DISCUSSION THREAD

**Discussion by Sreeragsrg77** [strimzi/strimzi-kafka-operator#9910] [General]
I was able to delete a topic from kafka cluster using offserexplorer ,akhq and with kubectl kafkatopic commands as well, but once it shows deleted within 2 mins same topics are again created.

Can anyone help me out with this issue ?

I am using offsetexplorer version 2.0 to connect the kafka cluster.

2024-04-02 09:20:47,01657 INFO  [ZkClient-EventThread-39-localhost:2181] ZkTopicsWatcher:127 - Topics deleted from ZK for watch 75: [test-1]
2024-04-02 09:20:47,56313 INFO  [ZkClient-EventThread-39-localhost:2181] ZkTopicsWatcher:143 - Topics created in ZK for watch 76: [test-1]

The above logs are from our kafka-cluster-entity-operator pods.

**Comment by scholzj** [comment]
I think you need to provide more details ...
* Did you disable topic auto-creation?
* What Strimzi / Topic OPerator version you use?
* How exactly did you created / deleted the topic?

And so on.

**Reply by Sreeragsrg77** [reply]
1. No, wasn't it disabled by default?
2.  topic-operator-0.36.1
3. Created using offsetexplorer and for deletion I have tried both offsetexplorer and kubectl  delete kafkatopics <topic name> -n <namespace> command


Our kafka cluster is in kubernetes ,FYI.

**Reply by scholzj** [reply]
> No, wasn't it disabled by default?

No, it has always been enabled by default in Apache Kafka. So this is where you should likely start.
